src/lib.rs

```diff
@@ -233,25 +233,20 @@ extern crate proc_macro;
 #[macro_use]
 extern crate quote;
 
-use crate::proc_macro::TokenStream;
-use proc_macro2::TokenStream as TokenStream2;
+use proc_macro::TokenStream;
+use proc_macro_error::{abort_call_site, proc_macro_error, set_dummy};
 use quote::ToTokens;
 
 mod analyze;
 mod attr;
-mod diag;
 mod gen;
 mod proxy;
-mod spanned;
-
-use crate::{
-    diag::SpanExt,
-    spanned::Spanned,
-};
-
 
 /// See crate documentation for more information.
+#[proc_macro_error]
 #[proc_macro_attribute]
+#[allow(unused)]
+#[allow(dead_code)]
```

Comment:
Why are these `allow` attributes necessary?

Reply:
Oops, these are not for this crate, I messed up :)